doc/mkdocs/docs/features/macros.md

```diff
@@ -63,6 +67,8 @@ The diagnostics messages can also be controlled with the CMake option `JSON_Diag
     #define JSON_DIAGNOSTICS 0
     ```
 
+The macro was introduced in version 3.10.0.
+
 ## `JSON_HAS_CPP_11`, `JSON_HAS_CPP_14`, `JSON_HAS_CPP_17`, `JSON_HAS_CPP_20`
```

Comment:
These macros aren't user-visible. They're `#undef`'ed in `macro_unscope.hpp`. Which raises the broader question: should internal macros be documented and specifically labeled as such? Also, there's `JSON_TEST_KEEP_MACROS` which does mean we can "leak" some internal macros.

Food for thought for a future documentation update…